next.clayui.com/content/docs/components/forms/css-checkbox-radio.md

```diff
@@ -1,20 +1,13 @@
 ---
-title: "Checkbox and Radio"
-description: "Checkboxes and radios provide users with different selection and activation tools."
+title: 'Checkbox and Radio'
+description: 'Checkboxes and radios provide users with different selection and activation tools.'
 ---
 
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
-- [Checkboxes and Radios](#checkboxes-and-radios)
-	- [Default (stacked)](#default-(stacked))
-	- [Inline](#inline)
-	- [Without labels](#without-labels)
-- [Custom Checkboxes and radios](#custom-checkboxes-and-radios)
-	- [Checkboxes](#checkboxes)
-	- [Radios](#radios)
-	- [Inline](#inline)
-	- [Disabled](#disabled)
+-   [Checkboxes and Radios](#checkboxes-and-radios) - [Default (stacked)](<#default-(stacked)>) - [Inline](#inline) - [Without labels](#without-labels)
```

Comment:
It would be nice to check the documents one by one that he modified in the documents, this breaks the ToC listing.

Reply:
Its probably gonna be difficult to go through each file one by one and running format. I think it might be best to run format first, and then fix any makdown we catch along the way or catch in this PR review. What do you think?

Reply:
Yeah, I can analyze all that ended up breaking if you wish and add in PR review. I just didn't want to pollute the PR with all the comments I might have.

Reply:
I can go through it first and I will ping you when I need anther set of eyes.

Reply:
ok, thanks Bryce